### ISSUE TITLE
NVDA no longer reading suggestions in Outlook @ mention list

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#14419]
### Steps to reproduce:
Open Outlook desktop
Open a new email
Navigate to the email body
Type @ and start typing a name 
Arrow up/down through the list when typing a name

### Actual behavior:
NVDA doesn't announce when suggestions are available or what the suggestions are when arrowing up/down.
Document  page 1  section 1  blank
down arrow
@qu
up arrow
@qu
down arrow
@quentin

### Expected behavior:
NVDA should read that suggestions are available, indexing (i.e. 1 of 5), and who each focused suggestion is. For example in the above case NVDA should read "Quentin"

### NVDA logs, crash dumps and other attachments:
Attached
#### NVDA installed/portable/running from source:
Installed and portable

#### NVDA version:
2023.3.1
#### Windows version:
22H2(OS Build 25158.1000)
#### Name and version of other software in use when reproducing the issue:
Outlook version: (Version 2211 Build 15806.20000)
#### Does the issue still occur after restarting your computer?
Yes
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does 
[nvda.log](https://github.com/nvaccess/nvda/files/10143555/nvda.log)
the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by michaelDCurran** [comment]
This has already been fixed by pr #14222 which is already in alpha builds, and will be officially available in NVDA 2023.1.
